src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -1042,6 +1042,15 @@ enum JavaThreadState {
   _thread_max_state         = 12  // maximum thread state+1 - used for statistics allocation
 };
 
+enum LockingMode {
+  // Use only heavy monitors for locking
+  MONITOR     = 0,
+  // Legacy stack-locking, with monitors as 2nd tier
+  LEGACY      = 1,
+  // New lightweight locing, with monitors as 2nd tier
```

Comment:
```suggestion
  // New lightweight locking, with monitors as 2nd tier
```
I was just passing by your PR. Pointing out a minor typo. Cheers!